grails-app/domain/org/pih/warehouse/core/Event.groovy

```diff
@@ -9,6 +9,7 @@
  **/
 package org.pih.warehouse.core
 
+import org.pih.warehouse.auth.AuthService
```

Comment:
Is this required?

Reply:
Not anymore, forgot to delete it in the latest commit.